.github/workflows/test-go.yml

```diff
@@ -6,10 +6,12 @@ on:
       - main
 
 jobs:
-  unit:
-    name: unit
-    runs-on: ubuntu-20.04
-    steps:
+  jobInstall:
```

Comment:
why not using a matrix strategy ?

Reply:
I've found the matrix strategy to use different version of go for the tests, how can be used to run different jobs with different tests ?